roscala/src/main/scala/coop/rchain/roscala/Vm.scala

```diff
@@ -53,11 +53,11 @@ object Vm {
   def execute(opcode: Opcode, globalEnv: GlobalEnv, state: State): Unit =
     opcode match {
       case OpAlloc(n) =>
-        state.ctxt.argvec = Tuple(new Array[Ob](n))
+        state.ctxt.argvec = new Tuple(new Array[Ob](n))
         state.nextOpFlag = true
 
       case OpPushAlloc(n) =>
-        val t = Tuple(new Array[Ob](n))
+        val t = new Tuple(new Array[Ob](n))
```

Comment:
Why don't we create a companion object to encapsulate `Tuple` creation into `apply` methods?